### PR TITLE
Adding HINTS.md to palindrome-products exercise

### PR DESCRIPTION
**Pull request by jmbradnan**
Adding HINTS.md to palindrome-products exercise (#209).    Let me know if hint makes sense or needs to be tweaked.   thx

### CODE REVIEW

File: exercises/palindrome-products/HINTS.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Hints
+
+For this exercise, you will need to create a set of factors using Tuples.
```
Comment (ErikSchierboom) [inline exercises/palindrome-products/HINTS.md]: Could you replace Tuples with tuples (lowercase)?

File: exercises/palindrome-products/HINTS.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+For more information on Tuples, see [this link](https://msdn.microsoft.com/en-us/library/system.tuple(v=vs.110).aspx).
```
Comment (ErikSchierboom) [inline exercises/palindrome-products/HINTS.md]: Could you replace Tuples with tuples (lowercase)?

File: exercises/palindrome-products/HINTS.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Hints
+
+For this exercise, you will need to create a set of factors using Tuples.
+
```
Comment (ErikSchierboom) [inline exercises/palindrome-products/HINTS.md]: Please remove this empty line.

**Comment by ErikSchierboom** [comment]
Thanks! :tada:

**Comment by jmbradnan** [comment]
@ErikSchierboom quick question for you - it looks like 10 commits were merged with this change (all of my previous ones), am I reading that correctly?     Those should have been merged already so want to make sure I am not doing something weird with my branches.

**Comment by ErikSchierboom** [comment]
@jmbradnan Well, actually it does look like you are doing something weird with your branches :) In this case, as those commits were already merged, it doesn't really matter. But, normally I would expect to see one commit in the PR. Having those other commits in the PR means that the branch on which the PR is based was not up-to-date with the latest `exercism/master` branch. 

Earlier, I suggested you do the following to create a new branch:

```bash
git checkout master
git fetch exercism
git reset --hard exercism/master
git checkout -b <your-branch>
```

Did you do that?

Another option is to do the following while being on your branch:

```bash
git fetch exercism
git rebase exercism/master
```

This will rebase your commits on the latest master, which is precisely what we want. However, rebasing is a bit of an advanced Git feature. You can read more about it here: https://git-scm.com/docs/git-rebase

**Comment by jmbradnan** [comment]
yes I did follow your first set of steps, I am really not sure what is happening seems pretty straightforward.   I was going to pick up one more of these, I will try rebase first and see if it makes a difference.

**Comment by ErikSchierboom** [comment]
@jmbradnan Hmmm, very weird. And what do you see when you enter: `git remote -v`?

**Comment by jmbradnan** [comment]
exercism        https://github.com/exercism/xcsharp.git (fetch)
exercism        https://github.com/exercism/xcsharp.git (push)
origin  https://github.com/jmbradnan/xcsharp.git (fetch)
origin  https://github.com/jmbradnan/xcsharp.git (push)

**Comment by ErikSchierboom** [comment]
Hmmm, looking good. Well, let's try again with the next PR :)

**Pull request merged**